### PR TITLE
fix #17 Error at line 31

### LINKED ISSUE #17: Error ocurred at line 31 [ too many arguments]
This script may happen to have an Error with line 31

./jenkins-backup.sh: line 31: [: 引数が多すぎます

Maybe this ocurre by 
if [ -f "$JENKINS_HOME/plugins/"*.hpi.pinned -o -f "$JENKINS_HOME/plugins/"*.jpi.pinned ] ; then

"$JENKINS_HOME/plugins/"*.jpi.pinned
this wild card line will extract many file list.

Like this:
"-o -f /var/lib/jenkins//plugins/ant.jpi.pinned /var/lib/jenkins//plugins/antisamy-markup-formatter.jpi.pinned /var/lib/jenkins//plugins/credentials.jpi.pinned /var/lib/jenkins//plugins/cvs.jpi.pinned /var/lib/jenkins//plugins/javadoc.jpi.pinned /var/lib/jenkins//plugins/junit.jpi.pinned /var/lib/jenkins//plugins/ldap.jpi.pinned /var/lib/jenkins//plugins/mailer.jpi.pinned /var/lib/jenkins//plugins/matrix-auth.jpi.pinned /var/lib/jenkins//plugins/matrix-project.jpi.pinned /var/lib/jenkins//plugins/maven-plugin.jpi.pinned /var/lib/jenkins//plugins/pam-auth.jpi.pinned /var/lib/jenkins//plugins/ssh-slaves.jpi.pinned /var/lib/jenkins//plugins/subversion.jpi.pinned /var/lib/jenkins//plugins/translation.jpi.pinned ']'"

to fix this. (just idea)
readonly hpi.pinned_count=$(find $JENKINS_HOME/plugins/ -name *.jhi.pinned | wc -l)
readonly jpi.pinned_count=$(find $JENKINS_HOME/plugins/ -name *.jpi.pinned | wc -l)
if [ $hpi.pinned_count -ne 0 -o jpi.pinned_count -ne 0 ]; then

よろしくお願いします。

### PR DESCRIPTION
**Pull request by ynott**

### CODE REVIEW

File: jenkins-backup.sh [hunk 1]
```diff
@@ -27,7 +27,9 @@ mkdir -p "$ARC_DIR/"{plugins,jobs,users}
 
 cp "$JENKINS_HOME/"*.xml "$ARC_DIR"
 cp "$JENKINS_HOME/plugins/"*.[hj]pi "$ARC_DIR/plugins"
-if [ -f "$JENKINS_HOME/plugins/"*.hpi.pinned -o -f "$JENKINS_HOME/plugins/"*.jpi.pinned ] ; then
+hpi_pinned_count=$(find $JENKINS_HOME/plugins/ -name *.jhi.pinned | wc -l)
```
Comment (sue445) [inline jenkins-backup.sh]: jhi ?

Comment (ynott) [inline jenkins-backup.sh]: Oops. 
I will fix it.

Comment (ynott) [inline jenkins-backup.sh]: Fix extension name.
Please check it.

**Comment by sue445** [comment]
I checked my jenkins and no problem!

**Pull request merged**